airflow-core/src/airflow/migrations/versions/0042_3_0_0_add_uuid_primary_key_to_task_instance_.py

```diff
@@ -31,6 +31,9 @@
 from sqlalchemy import text
 from sqlalchemy.dialects import postgresql
 
+BATCH_SIZE = 10000
```

Comment:
I think 10k is fine. Let's add a escapte hatch -- an env var to config if someone wanted -- a hind conf

```suggestion
BATCH_SIZE = int(os.environ.get("_AIRFLOW_MIGRATION_TI_ID_BATCH_SIZE", DEFAULT_BATCH_SIZE))
```

Reply:
I'm adding a config "migration_batch_size" that can be reused across all migrations